src/main/java/ch/njol/skript/localization/Message.java

```diff
@@ -82,7 +82,18 @@ public final String getValue() {
 		validate();
 		return value;
 	}
-	
+
+	/**
+	 * Gets the text this Message refers to. If value is null returns a default value.
+	 *
+	 * @param defaultValue The string this method refers to if value is null
+	 * @return This message's value or default value if null
+	 */
+	public final String getValueOrDefault(String defaultValue) {
```

Comment:
I don't agree with having a fallback for localization. If it doesn't exist don't print anything. Especially when it comes to the prefix, the Skript user can just remove the node and then no prefix would even be shown, prevent forced configurations.

Reply:
You do need the fallback, in the case where locales fail to load you must still notify the user of what went wrong.

Reply:
> You do need the fallback, in the case where locales fail to load you must still notify the user of what went wrong.

We could just leave it as `""` if it's unset, the original plan was to allow an empty value but people can easily do `<reset>` with nothing else in the lang. As for fail to load the message class already does that upon invalid entries whenever `validate()` gets called

so in truth while a fallback would be needed no matter what we could decide against a fallback for `[Skript]` and do `""`
would prefer more inputs before making this decision.

Reply:
Best to keep it as-is, the prefix is a crucial component of Skript and must work regardless of localization.

Reply:
It's a prefix, it's not crucial at all.

Reply:
A prefix is indeed important in cases where you get in game error messages or any message in general and you don't know where that came from. IMO a fallback to `[Skript]` is a good choice, users can use `<reset>` to make it empty as Fusezion suggested.